dhcpv4/server4/server_test.go

```diff
@@ -73,15 +73,15 @@ func setUpClientAndServer(t *testing.T, iface net.Interface, handler Handler) (*
 		IP:   loAddr,
 		Port: randPort(),
 	}
-	s, err := NewServer(&saddr, handler)
+	s, err := NewServer(iface.Name, &saddr, handler)
 	if err != nil {
 		t.Fatal(err)
 	}
 	go func() {
 		_ = s.Serve()
 	}()
 
-	clientConn, err := nclient4.NewIPv4UDPConn("", caddr.Port)
+	clientConn, err := dhcpv4.NewIPv4UDPConn("", caddr.Port)
```

Comment:
This should be `NewIPv4UDPConn("", caddr.Port)` instead of `dhcpv4.NewIPv4UDPConn("", caddr.Port)`.

Reply:
It is now in the same package.